roles/unmanic/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+#########################################################################
+# Title:            Community: unmanic                                  #
+# Author(s):        chazlarson                                          #
+# URL:              https://github.com/Cloudbox/Community               #
+# Docker Image(s):  josh5/unmanic                                       #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: "Setting CloudFlare DNS Record"
+  include_role:
+    name: cloudflare-dns
+  vars:
+    record: unmanic
+  when: cloudflare_enabled
+
+- name: Stop and remove any existing container
+  docker_container:
+    name: unmanic
+    state: absent
+
+- name: Create unmanic directories
+  file: "path={{ item }} state=directory mode=0775 owner={{ user.name }} group={{ user.name }}"
+  with_items:
+    - /opt/unmanic
+    - /mnt/local/unmanic
```

Comment:
I think all these type of locations need a standard. Standard _downloads_ location or standard temp _location_ (eg transcodes folder).

So perhaps you can put that where we put the emby/plex transcodes folder, just change the basename, whered you use `unmanic`.